cmd/cloudtrail/cloudtrail_test.go

```diff
@@ -34,25 +33,16 @@ func TestIgnoreListFilter(t *testing.T) {
 	var testUsername6 *string
 	testCloudTrailEvent6 := `{"eventVersion": "1.09","userIdentity": {"sessionContext": {"sessionIssuer": {"arn": ""}}}}`
 
-	TestLookupOutputs := []*cloudtrail.LookupEventsOutput{
-		{
-			Events: []types.Event{
-				{Username: &testUsername1, CloudTrailEvent: &testCloudTrailEvent1},
-				{Username: &testUsername2, CloudTrailEvent: &testCloudTrailEvent2},
-			},
-		},
-		{
-			Events: []types.Event{
-				{Username: &testUsername3, CloudTrailEvent: &testCloudTrailEvent3},
-				{Username: &testUsername4, CloudTrailEvent: &testCloudTrailEvent4},
-			},
-		},
-		{
-			Events: []types.Event{
-				{Username: &testUsername5, CloudTrailEvent: &testCloudTrailEvent5},
-				{Username: testUsername6, CloudTrailEvent: &testCloudTrailEvent6},
-			},
-		},
+	TestLookupOutputs := []types.Event{
```

Comment:
**chore:** rename the variable

Those are no longer event lookup outputs, but just events now.